src/systems/force_torque/ForceTorque.hh

```diff
@@ -50,6 +51,9 @@ namespace systems
     /// \brief Destructor
     public: ~ForceTorque() override;
 
+    /// Documentation inherited
+    public: System::PriorityType ConfigurePriority() final;
```

Comment:
The only issue I see with this API is what if  we want to configure `PreUpdate` and `Update` priorities separately. Perhaps we should provide two interfaces?

Reply:
at one point I had thought about being able to specify separate priority values for `PreUpdate` and `Update`, but to start with I opted for a simpler approach using a shared priority for each system. Looking through `src/systems`, I only see two systems that implement both `ISystemPreUpdate` and `ISystemUpdate`:

* [LinearBatteryPlugin](https://github.com/gazebosim/gz-sim/blob/main/src/systems/battery_plugin/LinearBatteryPlugin.hh#L76-L77)
* [PythonSystemLoader](https://github.com/gazebosim/gz-sim/blob/main/src/systems/python_system_loader/PythonSystemLoader.hh#L71-L72)

The `PythonSystemLoader` implements both in order to allow python systems to implement either callback, and I haven't looked into the details of `LinearBatteryPlugin`, but given that only one of our systems implements both `PreUpdate` and `Update`, I think it's reasonable to share the priority value, though we should open a feature request to configure priority for these callbacks separately

Reply:
I agree we should open a separate issue for this. I also mentioned to @scpeters at one point that it would be good to have a way to configure priorities separately, but upon further thought, it might make plugin configuration very hard. I think it's worth having the discussion, but I would not be in favor of adding it here.